oic.r.sp.raml

```diff
@@ -0,0 +1,67 @@
+#%RAML 0.8
+title: Security Profile Resource
+version: v1.0-20180508
+documentation:
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+schemas:
+  - SP: !include schemas/oic.r.sp.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+            enum: ["oic.if.baseline"]
+
+/oic/sec/sp:
+  description: |
+    Resource specifying supported and active security profile(s)
+
+  displayName: Security Profile Resource
+  is: [ interface ]
+
+  get:
+    description: |
+      Retrieves Security Profile data
+    responses:
+      200:
+        body:
+          application/json:
+            schema: SP
+            example: |
+              {
```

Comment:
can we have the rt value here as example?
the tooling otherwise will fail

Reply:
Did not add it, to be consistent with other Security RAML examples.
But if breaks the the tooling, can add it (and eventually to other security RAMLs too).

Reply:
To date, Sec WG hasn't been using the tooling to my knowledge.  We probably need to update all of the Security data models...